classes/services/queryBuilders/ContextQueryBuilder.inc.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * @file classes/services/QueryBuilders/ContextQueryBuilder.inc.php
+ *
+ * Copyright (c) 2014-2018 Simon Fraser University
+ * Copyright (c) 2000-2018 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class ContextQueryBuilder
+ * @ingroup query_builders
+ *
+ * @brief Journal list query builder
+ */
+namespace APP\Services\QueryBuilders;
+
+use Illuminate\Database\Capsule\Manager as Capsule;
+
+class ContextQueryBuilder extends \PKP\Services\QueryBuilders\PKPContextQueryBuilder {
+	/** @copydoc \PKP\Services\QueryBuilders\PKPContextQueryBuilder::$db */
+	protected $db = 'presses';
```

Comment:
I think at some point in the near future we could rename `presses` / `journals` to `contexts`, likewise `press_settings` / `journal_settings` to `context_settings`, etc. But let's leave that for later, and when it happens, this can get removed.

Reply:
:100: